src/payment/strategies/chasepay/chasepay.ts

```diff
@@ -42,7 +42,7 @@ export interface ChasePay {
         COMPLETE_CHECKOUT: ChasePayEventType.CompleteCheckout;
         CANCEL_CHECKOUT: ChasePayEventType.CancelCheckout;
     };
-    isChasePayUp(): boolean;
+    isChasePayUp: boolean;
```

Comment:
this is used as `if(chasePay.isChasePayUp)`

New Typescript version was complaining about this